typeset/mjnode.js

```diff
@@ -1,23 +1,14 @@
-const mjAPI = require('mathjax-node')
+let mjAPI;
 
-const convertMathML = async (log, mathMap/*: Map<string, {xml: string, fontSize: number} */, outputFormat) => {
+const startAPI = async (log) => {
+  mjAPI = require('mathjax-node')
```

Comment:
I'm interested in the reasoning behind moving this `require` into a function, which is typically something I've heard should be avoided.

Reply:
I can't remember why I did that :confused:  I'll move it out.

Reply:
Oh I remember now.  If you look in the `convertMathML` function in this file, the first thing it checks is whether `mjAPI` is defined.  I'm going to add a different variable called `mjStarted`, to make this clearer.